lib/brakeman/checks/check_cross_site_scripting.rb

```diff
@@ -115,7 +115,7 @@ def check_for_immediate_xss exp
           :warning_type => "Cross Site Scripting",
           :warning_code => warning_code,
           :message => message,
-          :code => match,
+          :code => code,
```

Comment:
You'll probably want to verify my assumption here. If `:code` *is* supposed to be `match` then the `code` assignment just above should be removed.